features/importers/venepaikka_harbors/importer.py

```diff
@@ -198,6 +226,24 @@ def _import_service_map_url(self, feature: Feature, servicemap_id: str):
                 defaults={"url": f"{self.servicemap_url}{servicemap_id}"},
             )
 
+    @staticmethod
+    def _import_harbor_details(feature: Feature, harbor_details: Mapping):
+        data = {}
+
+        if harbor_details["berth_moorings"]:
+            data["berth_moorings"] = list(set(harbor_details["berth_moorings"]))
+
+        if harbor_details["berth_depths"]:
+            data["berth_min_depth"] = min(harbor_details["berth_depths"])
```

Comment:
well, we do not know when these will be added. probably when we have a UI for harbors personnel, they might start adding those for some berths gradually. even the, I am pretty sure, there will be a very long period when some of the berths will have `null` depth, while other might have some value. not sure how this affects ahti in general, but this importer probably filters out `null`s here `berth_depths: properties.piers.edges[*].node.properties.berths.edges[*].node.berthType.depth[]`, right?

Reply:
Yeah, null values are ignored. Looks like the jmespath tool automatically ignores them.

Reply:
I added one berth with depth information into the faked/test response, but all others have depth as `null`.

Reply:
> jmespath tool seems to automatically ignore them
yeah, that's how I understood it 👍 

alright then!